Content.Client/Geras/GerasSystem.cs

```diff
@@ -0,0 +1,34 @@
+using Content.Shared.Humanoid;
+using Content.Shared.Geras;
+using Robust.Client.GameObjects;
+
+namespace Content.Client.Geras;
+
+public sealed class GerasSystem : SharedGerasSystem
+{
+    public override void Initialize()
+    {
+        base.Initialize();
+        SubscribeNetworkEvent<GerasChildEntity>(OnGerasChildEntity);
+    }
+
+    private void OnGerasChildEntity(GerasChildEntity ev)
+    {
+
+        var parentUid = GetEntity(ev.ParentUid);
+        var childUid = GetEntity(ev.ChildUid);
+
+        var skinColor = Color.Green;
+
+        if (!TryComp<HumanoidAppearanceComponent>(parentUid, out var appearanceComp))
+        {
+            return;
+        }
+        if (TryComp<SpriteComponent>(childUid, out var sprite))
+        {
+            skinColor = appearanceComp.SkinColor;
+            sprite.Color = skinColor;
+        }
+        return;
+    }
+}
```

Comment:
do not use network events for this, use `AppearanceComponent`

https://docs.spacestation14.com/en/ss14-by-example/making-a-sprite-dynamic.html

Reply:
I think I've done it right now?